script.skin.helper.service/plugin.py

```diff
@@ -46,8 +46,36 @@ def __init__(self):
             elif action == "LAUNCH":
                 xbmcplugin.setResolvedUrl(handle=int(sys.argv[1]), succeeded=False, listitem=xbmcgui.ListItem())
                 path = sys.argv[2].split("&path=")[1]
-                xbmc.executebuiltin("Action(Close)")
                 xbmc.executebuiltin(path)
+            elif action == "FOCUSANDCLICK":
+                #used as workaround to display local media in extendedinfo actorinfo
```

Comment:
Yes, that's why I hereby ask you if it can be added to extended info.
In the actorinfo dialog there are some containers: movie roles, tvshow roles, videos etc.
The request is 1 additional container/list: items in library

Yes, it is included in Titan